homeassistant/components/light/hue.py

```diff
@@ -394,3 +413,15 @@ def device_state_attributes(self):
         if self.is_group:
             attributes[ATTR_IS_HUE_GROUP] = self.is_group
         return attributes
+
+    def _async_update_callback(self):
```

Comment:
I think this should be annotated with `@callback` that you can import from core.py.